<launch>
    <!-- Start the odometry transformer node -->
    <node name="odometry_transformer_robot_x_toglobal" pkg="unity_ros_tf" type="odometry_fromodom_transformer.py" output="screen">
        <param name="parent_frame"  value="map" />
        <param name="child_frame" value="multijackal_02/odom" />
        <param name="z_offset_human" value="0.0" />
        <param name="z_offset_floor" value="0.0" />
        <remap from="/odom" to="/multijackal_02/odom"/>
        <remap from="/global_odom" to="/global/multijackal_02/odom"/>
        <param name="rosconsole_format" value="[${severity}] [${time}]: ${message}" />
        <param name="rosconsole_level" value="debug" />
    </node>

    <!-- <node name="odometry_transformer_robot_1_toglobal" pkg="unity_ros_tf" type="odomfromtf.py" output="screen">
        <param name="parent_frame"  value="map" />
        <param name="child_frame" value="multijackal_02/odom" />
        <param name="z_offset_human" value="0.0" />
        <param name="z_offset_floor" value="0.0" />
        <remap from="/odom" to="/multijackal_02/odom"/>
        <remap from="/global_odom" to="/global/test"/>
        <param name="rosconsole_format" value="[${severity}] [${time}]: ${message}" />
        <param name="rosconsole_level" value="debug" />
    </node> -->

    <!-- <node name="odometry_transformer_robot_2_toglobal" pkg="unity_ros_tf" type="odometry_transformer.py" output="screen">
        <param name="parent_frame"  value="map" />
        <param name="child_frame" value="multijackal_02/odom" />
        <param name="z_offset_human" value="0.0" />
        <param name="z_offset_floor" value="0.0" />
        <remap from="/global_odom" to="/global/test"/>
        <param name="rosconsole_format" value="[${severity}] [${time}]: ${message}" />
        <param name="rosconsole_level" value="debug" />
    </node> -->

    <!-- Start the occupancy grid transformer node
    <node name="occupancy_grid_transformer_node" pkg="unity_ros_tf" type="occupancy_grid_transformer.py" output="screen">
        <param name="parent_frame" value="map" />
        <param name="child_frame" value="multijackal_01/map" />
        <param name="origin_position_offset" value="[0.0, 0.0, 0.0]" />
        <param name="origin_orientation_offset" value="[0.0, 0.0, 0.0]" />
        <remap from="/robot_map" to="/multijackal_01/map"/> #name of original map
        <remap from="/global_map" to="/global/multijackal_01/map"/> #name of modified map
    </node> -->
</launch>
